### DISCUSSION TITLE
Cors v5

### DISCUSSION THREAD

**Discussion by ms1sistemas** [adonisjs/core#2401] [Help] [answered]
there is a configuration for CORS as it existed in v4.1 

thanks .....

**Comment by RomainLanz** [comment, marked as answer]
Hey @ms1sistemas! 👋 

The configuration should be at the same place in V5, `config/cors.ts`.
The default configuration can be found in the `@adonisjs/core` repository.

🔗 https://github.com/adonisjs/core/blob/develop/templates/config/cors.txt

**Reply by ms1sistemas** [reply]
tanks so much
